user/caribou/module/src/CLICTDEventConverter.cc

```diff
@@ -163,14 +163,18 @@ bool CLICTDEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Standa
 
       // Check for T0 signal going from high to low
       if((triggers & 0x1) && !(signals & 0x1)) {
-        t0_seen_++;
-        if(t0_seen_ == 1) {
-            EUDAQ_INFO("CLICTD: Detected 1st T0 signal directly: T0 flag at " + to_string(time) + "ns");
-            // Discard this event:
-            return false;
+        if (time <= 10) {
```

Comment:
what was `time` again?

Reply:
`time` is a timestamp of the event. A falling edge of T0 that reset the counter has a counter value of 1 corresponding to timestamp 10 ns. If the timestamp is higher than 10 ns, that T0 event did not reset the counter. We still record such an event just to have the data available.

Reply:
Sounds good!